src/test/java/edu/kis/powp/jobs2d/events/SelectCommandTransformationScaleOptionListener.java

```diff
@@ -2,16 +2,16 @@
 
 import edu.kis.powp.jobs2d.command.ICompoundCommand;
 import edu.kis.powp.jobs2d.command.manager.DriverCommandManager;
-import edu.kis.powp.jobs2d.command.transformation.CommandTransformation;
-import edu.kis.powp.jobs2d.command.transformation.CommandTransformationScale;
+import edu.kis.powp.jobs2d.command.transformation.TransformationScale;
+import edu.kis.powp.jobs2d.command.visitor.CommandVisitorTransformation;
 import edu.kis.powp.jobs2d.features.CommandsFeature;
 
 import java.awt.event.ActionEvent;
 import java.awt.event.ActionListener;
 
 public class SelectCommandTransformationScaleOptionListener implements ActionListener {
-    double scaleFactorX;
-    double scaleFactorY;
+    private final double scaleFactorX;
+    private final double scaleFactorY;
 
```

Comment:
Could all command transformation listeners could be implemented as one listener (injecting trasformation instead of particular transformation params)